docs/commands/validate.asciidoc

```diff
@@ -0,0 +1,43 @@
+[[validate]]
+== validate
+
```

Comment:
New command that just runs validate (no cleaning) with josm validators on a map and adds tags for the errors. This will be useful for #3652 when we integrate validation into our unit test checks, but for now probably isn't something that needs to be exposed to the UI unless any users tell us they want to run just validation from the UI.